contrib/azure-setup.sh

```diff
@@ -49,13 +49,13 @@ showhelp() {
 requirements() {
     found=0
 
-    azureversion=$(azure -v)
+    azureversion=$(az -v)
     if [ $? -eq 0 ]; then
         found=$((found + 1))
         echo "Found azure-cli version: $azureversion"
     else
-        echo "azure-cli is missing. Please install azure-cli from"
-        echo "https://azure.microsoft.com/en-us/documentation/articles/xplat-cli-install/"
+        echo "azure cli is missing. Please install azure cli from"
+        echo "https://az.microsoft.com/en-us/documentation/articles/xplat-cli-install/"
```

Comment:
This isn't the XPlat CLI, this it the Azure CLI 2.0.

Reply:
My mistake, I changed the link to the appropriate one.